CMakeLists.txt

```diff
@@ -264,6 +253,28 @@ if (NOT EMSCRIPTEN)
     target_link_libraries(wasm-interp m)
   endif ()
 
+  # wastfmt
+  add_executable(wastfmt src/exe/wastfmt.c)
```

Comment:
OK, I was trying to think of something clever for removing sugar (`nutra-wast`, `wast-zero`, `diet-wast`). But maybe just `wast-desugar` or `wast-canonicalize`?